### PR TITLE
Basic handling for conversation streaming + conversation mutate()

### PR DESCRIPTION
**Pull request by lasryaric**

### CODE REVIEW

File: front/components/assistant/conversation/Conversation.tsx [hunk 1]
```diff
@@ -8,11 +10,28 @@ export default function Conversation({
...
+  } = useConversation({
+    conversationId,
+    workspaceId: owner.sId,
+  });
+
+  useEffect(() => {
+    const es = new EventSource(
+      `/api/w/${owner.sId}/assistant/conversations/${conversationId}/events`
+    );
+    es.onmessage = () => {
```
Comment (spolu) [inline front/components/assistant/conversation/Conversation.tsx:27]: Need a bit of filtering of message by ids :)

File: front/components/assistant/conversation/Conversation.tsx [hunk 2]
```diff
@@ -8,11 +10,28 @@ export default function Conversation({
...
+      void mutateConversation();
+    };
+
+    return () => {
```
Comment (spolu) [inline front/components/assistant/conversation/Conversation.tsx:31]: This closure is executed if useEffect triggres right?

File: front/components/assistant/conversation/Conversation.tsx [hunk 3]
```diff
@@ -8,11 +10,28 @@ export default function Conversation({
...
+    workspaceId: owner.sId,
+  });
+
+  useEffect(() => {
+    const es = new EventSource(
+      `/api/w/${owner.sId}/assistant/conversations/${conversationId}/events`
+    );
+    es.onmessage = () => {
+      void mutateConversation();
+    };
```
Comment (spolu) [inline front/components/assistant/conversation/Conversation.tsx:29]: What happens on time out? We need to handle reconnection no?

File: front/components/assistant/conversation/Conversation.tsx (hunk 2 above)
Comment (lasryaric) [reply front/components/assistant/conversation/Conversation.tsx:57]: Just seeing that. This closure is executed when this useEffect is re-triggered or if the component is unmounted.

File: front/components/assistant/conversation/Conversation.tsx (hunk 3 above)
Comment (lasryaric) [reply front/components/assistant/conversation/Conversation.tsx:55]: Just seeing that. I added re-connection yes.

**Review by lasryaric — COMMENTED** [review]
Ready for review @spolu

**Review by fontanierh — DISMISSED** [review]
LGTM 👍

File: front/components/assistant/conversation/Conversation.tsx [hunk 4]
```diff
@@ -9,11 +15,56 @@ export default function Conversation({
...
+  const {
+    conversation,
+    isConversationError,
+    isConversationLoading,
+    mutateConversation,
+  } = useConversation({
+    conversationId,
+    workspaceId: owner.sId,
+  });
+  // state use to re-connect to the events stream.
```
Comment (spolu) [inline front/components/assistant/conversation/Conversation.tsx:27]: nit: Please use sentences as comments as much as possible: Starts with a upper-case finish with `.`

**Review by spolu — DISMISSED** [review]
Left a small nit on comment format otherwise LGTM!

**Pull request merged**